cms/static/require-config.js

```diff
@@ -27,6 +27,7 @@ require.config({
         "jquery.immediateDescendents": "coffee/src/jquery.immediateDescendents",
         "datepair": "js/vendor/timepicker/datepair",
         "date": "js/vendor/date",
+        "moment": "js/vendor/moment.min",
```

Comment:
I am concerned about adding a new dependency for what is ultimately a nice-to-have, cosmetic addition.

I read through the comments earlier you wrote about how to handle the timestamp and I'm not sure the best way to handle that. It might be to follow the pattern that the due date and other times use - going through the server backend to make sure the datestamp is right or something like that. @singingwolfboy could you take a look at this PR and weigh in?